### PR TITLE
Add jupyter-server-proxy for Tensorboard

### PR DESCRIPTION
**Pull request by VaishnaviHire**
- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message
- [x] JIRA link(s):  https://issues.redhat.com/browse/RHODS-2978
- [x] The Jira story is acked
- [ ] An entry has been added to the latest build document in [Build Announcements Folder](https://drive.google.com/drive/folders/1sgkK1WZgGo9CXsLizNe0GbAzVKuSKrZL).
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious)

### CODE REVIEW

**Comment by VaishnaviHire** [comment]
Tested on OCP 4.10

For Notebook-controller: 

Screen Shot 2022-07-18 at 7 00 57 PM

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ check:
   - thoth-precommit
   - thoth-build
 build:
-  base-image: "quay.io/thoth-station/s2i-minimal-notebook:latest"
+  base-image: "quay.io/thoth-station/s2i-minimal-py38-notebook:v0.3.0"
```
Comment (lugi0) [inline .aicoe-ci.yaml:4]: wasn't pytorch's base image the minimal CUDA one? or is this something else?

File: Pipfile [hunk 2]
```diff
@@ -38,10 +38,19 @@ scipy = "==1.6.2"
 seaborn = "*"
 sqlalchemy = "*"
 statsmodels = "*"
-tensorboard = "==1.15.0"
+tensorboard = "==2.6"
```
Comment (lugi0) [inline Pipfile:31]: If we change the tensorboard version then the metadata of the image should be updated to reflect it in the spawner

File: .aicoe-ci.yaml (hunk 1 above)
Comment (VaishnaviHire) [reply .aicoe-ci.yaml:4]: This update is just for the `aicoe-ci` pipeline. The base image would still be from https://github.com/red-hat-data-services/odh-deployer/blob/main/jupyterhub/cuda-11.4.2/manifests.yaml#L37.

cc: @harshad16

Comment (harshad16) [reply .aicoe-ci.yaml:4]: yes correct just for testing, as mentioned here as well. https://github.com/red-hat-data-services/s2i-tensorflow-gpu-notebook/pull/11#discussion_r924515915

File: .aicoe-ci.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ check:
   - thoth-precommit
```
Comment (harshad16) [inline .aicoe-ci.yaml]: can we remove this, it not very effective anymore.

File: Pipfile [hunk 4]
```diff
@@ -6,24 +6,24 @@ verify_ssl = true
...
+#jupyter-bokeh = "*"
 jupyter-server = "==1.8.0"
-jupyter-tensorboard = "*"
-jupyterlab = "==3.2.4"
-jupyterlab-git = "==0.30.0"
-jupyterlab-s3-browser = "*"
+#jupyter-tensorboard = "*"
+#jupyterlab = "==3.2.4"
+#jupyterlab-git = "==0.30.0"
+#jupyterlab-s3-browser = "*"
```
Comment (harshad16) [inline Pipfile]: lets remove these commented one out.
it would just create confusing.

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
just few comments

**Review by harshad16 — COMMENTED** [review]
/lgtm
thanks :100:

**Comment by VaishnaviHire** [comment]
@lugi0 Live Build that includes changes in Pytorch , Tensorflow images and updated dependency versions

quay.io/modh/rhods-operator-live-catalog:1.14.0-rhods-2978

**Comment by lugi0** [comment]
Tested and working well, hold on merging until https://github.com/red-hat-data-services/ods-ci/pull/470 is merged

**Review by lugi0 — APPROVED** [review]
Automation is merged, this can move ahead

**Review by samuelvl — APPROVED** [review]

**Pull request merged**